starfish/intensity_table/concatenate.py

```diff
@@ -0,0 +1,34 @@
+from typing import Iterable
+
+import xarray as xr
+
+from starfish.types import Features
+from .intensity_table import IntensityTable
+
+
+def concatenate(intensity_tables: Iterable[IntensityTable]) -> IntensityTable:
+    """Concatenate IntensityTables produced for different fields of view or across imaging rounds
+
+    IntensityTables are concatenated along the features axis, and the resulting arrays are stored
+    densely, even if the underlying data is sparse, since xarray does not yet support sparse array
+    structures. This means that spots that are identified in different rounds and channels will
+    be identified as separate features, even if they have exactly identical coordinates.
+
+    To merge spots that share coordinates across rounds and channels into single features amenable
+    to decoding, use IntensityTable.combine_first()
```

Comment:
How come you didn't want to just add IntensityTable.combine_first() at the end of this method?

Reply:
Because I don't think we really need that use case -- I would advocate for a separate function, otherwise it could be a bit confusing what this is designed to do.